app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java

```diff
@@ -44,6 +46,7 @@
     private ContainerMessagesViewModel mContainerMessagesViewModel;
     private MessageMapEntitiesViewModel mMessageMapEntitiesViewModel;
     private UsersLocationViewModel mUserLocationViewModel;
+    private Handler mBackgroundHandler;
```

Comment:
Its used for different "off the ui-thread" operations and therefore the name backgroundHandler